### ISSUE TITLE
Hide connections and datasets for user with role "viewer"

### ISSUE THREAD

**Issue by Alex-FIR-IT** [datalens-tech/datalens#263]
Is there any way to hide connections and datasets in /workbooks/ for a user with a role "Viewer"? I don't want to show any information about my database.
![datalens1](https://github.com/user-attachments/assets/639a1c60-c327-4610-9ac9-fef4043d7477)

**Comment by blshark2008** [comment]
Добрый день! Вопрос актуален. Возможно ли скрыть наборы данных для пользователя? Чтобы сразу грузился дашборд нужный, а настройки были только под администратором?

**Comment by Maksosmart** [comment]
Согласен, тоже волнует этот вопрос, роль Viewer своим наименованием наталкивает на возможность исключительно просмотра материалов собранных чартов, дашбордов, но никак не "служебной" информации о подключениях и датасетах
